index.html

```diff
@@ -16,7 +16,8 @@
         THE RIGHT DATA TO THE RIGHT PEOPLE AT THE RIGHT TIME.
       </div>
       <div class="ds-u-font-size--base ds-u-padding-y--3">
-        The Beneficiary Claims Data API (BCDA) enables Accountable Care Organizations (ACOs) to retrieve Medicare Part A, Part B, and Part D claims data for their assigned beneficiaries. BCDA is currently serving ACOs participating in the Medicare Shared Savings Program and other eligible CMS Alternative Payment Models.
+        The Beneficiary Claims Data API (BCDA) gives organizations the insights they need to provide high quality, coordinated
```

Comment:
Assume this should stay centered, but in the wireframe it's left-aligned so would want to quickly confirm with Ali 😄